package.json

```
@@ -43,6 +43,6 @@
     "solidity-coverage": "^0.7.17"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=16.15.1 <17"
```

Using the latest v16 release seems too restrictive since most people don't update for every patch?
```suggestion
    "node": ">=16 <17"
```

---

Agree.